org/diceresearch/nebula/orchestrator.py

```diff
@@ -25,6 +27,11 @@ def goNextLevel(identifier):
     EVIDENCE_RETRIEVAL_RESULT = current[8]
     STANCE_DETECTION_RESULT = current[10]
 
+    #notification
+    REGISTRATION_TOKEN = current[14]
+    NOTIFICATION_TITLE = "Your result is ready!"
+    NOTIFICATION_BODY = "Your result is now available.",
```

Comment:
I guess it would be good if we include the ID of the request, so that people know how they can access the answer.

We may also want to ask our partner what we should send. :thinking: